.github/workflows/federated-integ-test.yml

```diff
@@ -169,12 +169,12 @@ jobs:
           cd lte/gateway
           export MAGMA_DEV_CPUS=3
           export MAGMA_DEV_MEMORY_MB=9216
-          fab federated_integ_test:build_all=False,orc8r_on_vagrant=True
+          fab federated-integ-test --orc8r-on-vagrant
       - name: Get test results
         if: always()
         run: |
           cd lte/gateway
-          fab get_test_summaries:dst_path="test-results"
+          fab get-test-summaries --integration-tests --sudo-tests --dst-path="test-results"
```

Comment:
--sudo-tests tries to fetch test results from the magma vm - this was not done before. should be false, right?

Reply:
The default before was `True` for both, which is the same here.
Function head from master:
```
def get_test_summaries(
        gateway_host=None,
        test_host=None,
        dst_path="/tmp",
        integration_tests='True',
        sudo_tests='True',
        dev_vm_name="magma",
):
```

Reply:
:+1: you are right. If you like, you can remove --sudo-tests here. It's not really hurting - just never returns something.